CMakeLists.txt

```diff
@@ -13,7 +13,7 @@
 cmake_minimum_required(VERSION 2.8.10 FATAL_ERROR)
 
 # explicitly set certain policies
-cmake_policy(VERSION 2.8.10)
+cmake_policy(VERSION 2.8.12)
```

Comment:
Will this work? We require V2.8.10 as a minimum, which when used doesn't know anything about the policies of V2.8.12.